packages/ibm-products/src/components/Datagrid/Datagrid/DatagridHeaderRow.tsx

```diff
@@ -244,6 +247,8 @@ const HeaderRow = (
                   [`${blockClass}__header-actions-column`]: header?.isAction,
                   [`${blockClass}__with-slug`]:
                     header.slug && React.isValidElement(header?.slug),
```

Comment:
can be removed and as we are going to apply aiLable corresponding classname on presence of slug prop

```suggestion
```

Reply:
Since some adapters are already using the existing slug classes, completely changing the classnames would break their code. Hence a separate class is added for AiLabel. Once we stop the support for slug completely, we could just remove the slug classes. For now it would be safer to keep separate classes.